packages/devtools_app/test/inspector_screen_test.dart

```diff
@@ -25,7 +23,7 @@ void main() {
   const screen = InspectorScreen();
 
   FakeServiceManager fakeServiceManager;
-  FakeServiceExtensionManager fakeExtensionManager;
+  FakeServiceExtensionManager? fakeExtensionManager;
```

Comment:
It is confusing to me why `FakeServiceManager fakeServiceManager;` isn't giving an analysis error here. If it isn't, do you know why that is?

Regardless, let's make both of these late